### PR TITLE
Added AISpawner and DayController

### PR DESCRIPTION
**Pull request by Lunium**

### CODE REVIEW

**Comment by YWAMWeessies** [comment]
Change Customer to Adventures to lessen confusion

File: behind-the-tankard/Assets/Scripts/AISpawner.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AISpawner : MonoBehaviour
+{
+    public DayController dayController;
+    public List<Vector3> spawnPoints;
```
Comment (MattijnP) [inline behind-the-tankard/Assets/Scripts/AISpawner.cs]: Having a list of vectors as input is not a friendly way of defining point in the world.
Its easier for designers to drag GameObjects around than to input world coordinates.
My suggestion is that you treat the child objects of the spawner as spawn points.
Designers can then create empty game objects inside the spawner and drag them around the world.
Well done on the gizmos!

File: behind-the-tankard/Assets/Scripts/AISpawner.cs [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+    public float maxSpawnDelay;
+    public float minSpawnDelay;
+    public float noiseRange;
+
+    void Start() {
+        StartCoroutine(Spawner());
+    }
+
+    private IEnumerator Spawner()
```
Comment (MattijnP) [inline behind-the-tankard/Assets/Scripts/AISpawner.cs]: Is there a way to stop the spawner?

File: behind-the-tankard/Assets/Scripts/AISpawner.cs [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    private void SpawnAdventurer(GameObject adventurer) 
+    {
+        Vector3 randomSpawnPoint = spawnPoints[Random.Range(0, spawnPoints.Count)];
+        Instantiate(adventurer, randomSpawnPoint, Quaternion.identity);
+    }
+
+    private float CalculateWaitTime(float busynessFactor)
+    {
+        float noise = Random.Range(-noiseRange, noiseRange);
+        return (maxSpawnDelay + (minSpawnDelay - maxSpawnDelay) / 100 * busynessFactor) + noise;
```
Comment (MattijnP) [inline behind-the-tankard/Assets/Scripts/AISpawner.cs]: Use `Mathf.Lerp(a, b, t)` for linear interpolation as it is easier to read.

Comment (MattijnP) [inline behind-the-tankard/Assets/Scripts/AISpawner.cs]: Use `Mathf.Lerp(a, b, t)` for linear interpolation as it is easier to read.

**Pull request merged**